flink-python/docs/reference/pyflink.table/expressions.rst

```diff
@@ -226,6 +227,8 @@ advanced type helper functions
     Expression.element
     Expression.array_contains
     Expression.array_distinct
+    Expression.map_keys
+    Expression.map_values
 
```

Comment:
i suggest map_from_arrays should in same location with Expression.map_keys/Expression.map_values.
Because they belong to the same  things, they shouldn't just be in lexicographical order

Reply:
`map_from_arrays` is not a function, is is expression defined in `pyflink/table/expressions.py` 
Expressions from there are defined in a separate section. That's the reason